src/book.cpp

```diff
@@ -161,7 +161,7 @@ void Book::updateFromOpds(const pugi::xml_node& node, const std::string& urlHost
   m_language = VALUE("language");
   m_creator = node.child("author").child("name").child_value();
   m_publisher = node.child("publisher").child("name").child_value();
-  m_date = fromOpdsDate(VALUE("updated"));
+  m_date = fromOpdsDate(VALUE("dc:issued"));
```

Comment:
Maybe we could add a fallback to `updated` if `dc::issued` if we have new library reading old opds stream.

Reply:
Done